db/index.js

```diff
@@ -67,7 +67,7 @@ const Review = sequelize.define('Review', {
       key: 'id'
     }
   },
-}, { timestamps: true });
+}, { timestamps: false });
```

Comment:
Is the date/time being kept track of another way? Or is it not relevant and you're just going to pull the latest records via index?

Reply:
Correct. As of right now I will just be using the primary key to determine the order that the reviews "came in" and use that when deciding what order they should be displayed.